README.md

```diff
@@ -19,27 +19,24 @@ You can view detailed documentation and guides at [https://ray-project.github.io
 
 ## Quick Start
 
-### Use Yaml
+### Use YAML
 
-Please choose the version you like to install. We will use nightly version `master` as an example
+Please choose the version you would like to install. The example below uses the nightly version `master`.
 
 | Version  |  Stable |  Suggested Kubernetes Version |
 |----------|:-------:|------------------------------:|
 |  master  |    N    | v1.19 - v1.24 |
 |  v0.3.0  |    Y    | v1.19 - v1.24 |
 
 ```
-export KUBERAY_VERSION=v0.3.0
+export KUBERAY_VERSION=master
```

Comment:
em. I still prefer to leave the stable version here. WDYT?

Reply:
Makes sense! I updated the example.